src/Executable.js

```
@@ -105,6 +111,63 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        if (this._maxBackoff != null && minBackoff >= this._maxBackoff) {
```

This should throw an error if `minBackoff ` is null.